packages/uikit/src/components/Link/ScanLink.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement, useMemo } from "react";
+import Link from "./Link";
+import OpenNewIcon from "../Svg/Icons/OpenNew";
+import BscScanIcon from "../Svg/Icons/BscScan";
+import { LinkProps } from "./types";
+
+interface ScanLinkProps extends Omit<LinkProps, "external" | "showExternalIcon"> {
+  icon?: ReactElement;
+  chainId?: number;
+}
+
+const icons: { [key: number]: ReactElement } = {
+  56: <BscScanIcon />,
```

Comment:
Let's use `ChainId` enum instead of number literal

Reply:
Let's consider aptos as a special case. Since this prop is optional so for aptos we simply don't pass it